spec/factories/virtual_template.rb

```diff
@@ -0,0 +1,16 @@
+FactoryGirl.define do
+  factory :virtual_template, :class => 'ManageIQ::Providers::CloudManager::VirtualTemplate' do
+    vendor 'amazon'
+    name 'vt'
+    location 'us-west-1'
+    ems_ref 'i-12345'
+    availability_zone_id 0
+    cloud_subnet_id 1
+    cloud_network_id 2
```

Comment:
With regard for the id numbers, prefer not to hardcode values.  This can hide (or even create) problems, in particular with the use of alternate REGION values.  i.e. 1 is not a valid id in REGION 1 where ids are in the 1-trillion range.